rfcs/OperationExpressions.md

```diff
@@ -0,0 +1,391 @@
+# RFC: Operation Expressions
+
+(WORKING TITLE!)
+
+**Proposed by:** [Benjie Gillam](https://twitter.com/benjie) - Graphile
+
+In the [Schema Coordinates RFC](./SchemaCoordinates.md) Mark introduced the
+concept of "schema coordinates" which give a standard human- and
+machine-readable way to unambiguously refer to entities within a GraphQL schema:
+types, fields, field arguments, enum values, directives and directive arguments.
+The scope of that RFC is deliberately very tight, and it serves that goal well,
+providing a one-to-one mapping between the schema coordinates and the schema
+entities.
+
+This RFC is to gather feedback on expansions of the Schema Coordinate syntax
+that could be used for different purposes whilst maintaining familiarity.
+
+## Aim
+
+The aim of this RFC is to give the GraphQL community a standard syntax that
+people, tools and documentation can use to concisely and consistently reference
+GraphQL operation concepts such as paths that is more fluid, expressive, and
+contains more context than the Schema Coordinates RFC that this RFC builds on
+top of.
+
+This is not intended to be a replacement of the Schema Coordinates RFC, but an
+extension to it for a number of additional use-cases.
+
+## Use cases
+
+#### Referencing a position within a GraphQL Operation Document
```

Comment:
pedantic question: should this be "Referencing a position within a GraphQL Query"?

asking cos what if the document contains two operations? does the operation expression encode which operation in the document it refers to? (can it / should it?)

Reply:
Yes it can, via the name:

```
OperationName:mutation>createFoo>foo>id
```